### ISSUE TITLE
TOR Panel Code Miner

### ISSUE THREAD

**Issue by anbu-squad** [UnamSanctam/SilentCryptoMiner#2215]
The task arose to configure the panel to connect via TOR
In general, a simple domain can be disabled - (104) Connection reset by peer

So I understand the code for sending requests to the panel is located directly in the miners?
can I see the code for sending requests to the panel?
I want to understand the logic and add new functionality

**Comment by anbu-squad** [comment]
in general, the idea is to load the Tor client as necessary in case of loss of control over the main domain

However, the question is actually how to download the Tor client
as an option, add a client download link to the pastebin type configuration

**Comment by anbu-squad** [comment]
You can use public Tor gateways (onion.ly, onion.foundation), then you won't have to pull up the client

**Comment by UnamSanctam** [comment]
Yes it uses cpp-httplib for communicating, if you want to use Tor then you'll need to download the Tor proxy and then run/install it. Then after that the client would need to be configured to use the proxy when accessing sites. Or if you want to sue onion.ly or onion.foundation then you can already use those in the current miner of course.

**Comment by anbu-squad** [comment]
I want to add a separate table of bots to the panel
but I can’t figure out how to fill it out correctly

So, I understand that two miners send post requests separately?


then, do we need to somehow implement a new sending of a post request?

**Comment by UnamSanctam** [comment]
I guess this also has to do with your new issue, but yes the injected miners are independent and send their own requests, though since they have the same computer ID (Unique Hash) then you can always group them together.
